.github/workflows/build.yaml

```diff
@@ -81,19 +81,75 @@ jobs:
           username: ${{ secrets.QUAY_STACKROX_IO_RW_USERNAME }}
           password: ${{ secrets.QUAY_STACKROX_IO_RW_PASSWORD }}
 
-      - name: Build and push OCI image
+      - name: Build and push OCI amd64 image
         if: github.event_name != 'pull_request'
         uses: docker/build-push-action@v5
         with:
           context: .
           push: ${{ github.event_name != 'pull_request' }}
-          tags: ${{ steps.meta.outputs.tags }}
+          tags: quay.io/stackrox-io/image-prefetcher:${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}-amd64
```

Comment:
Hm, this is quite a mouthful, but if there is no other way to get the tag...

Reply:
We can probably take the 1st tag from `steps.meta.outputs.tag` and would looks something like 
```
tags: ${{ fromJSON(steps.meta.outputs.json).tags[0] }}-amd64
```

Reply:
How about this approach instead:
- https://github.com/jdao-rh/image-prefetcher/pull/1
- https://github.com/stackrox/image-prefetcher/pull/49

Reply:
Sorry for late reply. lgtm. Will merge and runt the CI